### DISCUSSION TITLE
🏢 🕙  MicroMDM office hours

### DISCUSSION THREAD

**Discussion by jessepeterson** [micromdm/micromdm#763] [Announcements]
The MicroMDM project holds regular office hours 🏢 🕙  — regularly scheduled time for community members, contributors, interested parties and everybody else to ask questions, share ideas, discuss projects and progress, and otherwise connect and meet with fellow community members.

* Schedule: Every 2 weeks from [10am to 11am Pacific time](https://dateful.com/convert/pacific-time-pt?t=10am) on Thursdays
* Started: June 24, 2021
* Next few dates (updated ad hoc):
  * Jan 15, 2026
  * Jan 29, 2026
  * Feb 12, 2026
  * Feb 26, 2026
* [Google Meet link](https://meet.google.com/taw-kadv-hdn) (Updated 2025-01-15)
* [Notes Doc](https://docs.google.com/document/d/1sOHuBsKSwbHvc7EVmW0Jtk-uhIoYN8Q989OC17gFkyU/edit?usp=sharing)
* [Google Calendar](https://calendar.google.com/calendar/u/0?cid=ZTI2YzdiYzI5YjI4YTE1NTkzNDJmNTRmZmE5ZGZjMTYwZDkzZTQ4YmRlMzNlMTg3MDkzYmFjZDhhODhhMzVhMkBncm91cC5jYWxlbmRhci5nb29nbGUuY29t)

Please also refer to the MicroMDM project [Code of Conduct](https://github.com/micromdm/.github/blob/main/CODE_OF_CONDUCT.md).

**Comment by leojh** [comment]
Are these still being held? When is the next one?

**Reply by leojh** [reply]
never mind. I just saw Slack that they're still on.
